### DISCUSSION TITLE
<C-^> not recognized in mappings

### DISCUSSION THREAD

**Discussion by maxdarling** [JetBrains/ideavim#618] [Q&A]
When running `:nnoremap r <C-^>` in IntelliJ, nothing happens. 
But mapping other control-key sequences works just fine, e.g. `:nnoremap r <C-d>`
And pressing Ctrl+6 in normal mode in IntelliJ swaps to the alternate buffer, as expected.

So it seems like the mapping doesn't understand the character sequence `<C-^>`?

(I'm currently using the following workaround `:nnoremap r :e #<CR>`, so this isn't super pressing.)

**Comment by AlexPl292** [comment]
Hi, we had some issues with `<C-^>`. Could you please try `<C-S-6>`?
